### PR TITLE
Editorial update index.html: use JsonSchema and not JsonSchema2023

### PR DESCRIPTION
**Pull request by iherman**
This is just a leftover of the naming/versioning change around JSON Schemas.

(Editorial change, only in the examples).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1225.html" title="Last updated on Aug 4, 2023, 4:11 AM UTC (8b2e277)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1225/892eeac...8b2e277.html" title="Last updated on Aug 4, 2023, 4:11 AM UTC (8b2e277)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1731,7 +1731,7 @@ <h4>Representing Time</h4>
 only moment in time that is affected is the one second period of the leap second
 itself. That is, the moment after the most recent leap second can always be
 represented as the first moment of the next day (for example,
-`2023-01-01T00:00:00Z`), regardless of whether the system in question
+`-01-01T00:00:00Z`), regardless of whether the system in question
```
Comment (decentralgabe) [inline index.html:1734]:
```suggestion
`2023-01-01T00:00:00Z`), regardless of whether the system in question
```

**Review by decentralgabe — APPROVED** [review]
thanks!

**Review by andresuribe87 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]
Assuming suggestions are applied

**Review by dmitrizagidulin — APPROVED** [review]
(The Ecosystem Compatibility section seems to be an artifact of stray commit history)

**Review by msporny — CHANGES_REQUESTED** [review]
PR includes commits on main, needs a rebase.

**Comment by iherman** [comment]
> PR includes commits on main, needs a rebase.

@msporny I do not understand this remark. Rebase on what?

**Comment by msporny** [comment]
> @msporny I do not understand this remark. Rebase on what?

Look at the file diff... it contains your changes and a bunch of changes on an invalid branch on main. There was a PR that was pulled in a few days ago (the Ecosystem Compatability PR) that destroyed history, so I had to re-create history on main and force push to main, which then caused this PR (and many others) to have issues. To rebase:

```
git checkout main
git pull
git checkout iherman-patch-1
git rebase main
```

^^ That might do the trick, or you might have conflicts you need to resolve.

I can also squash-merge, but the current diff is probably going to confuse people that are reviewing, and they'll start suggesting changes to things that were not in this PR.

**Comment by iherman** [comment]
Moving over to #1229

**Comment by TallTed** [comment]
> Moving over to https://github.com/w3c/vc-data-model/pull/1229; this PR should be closed without merge.

As with #1226, maybe change title so people don't get drawn down the glitch rabbit hole?

**Comment by brentzundel** [comment]
When #1229 is merged, this PR should be closed.

**Review by awoie — CHANGES_REQUESTED** [review]
contains ecosystem compatibility which imo is not desired

**Comment by brentzundel** [comment]
@awoie this PR has been superseded by #1229 and will be closed when that PR is merged.

**Comment by msporny** [comment]
Merged via #1229, closing.

**Pull request closed without merging**